inc/inventory.class.php

```diff
@@ -411,43 +411,26 @@ function reportHbarWindows($config = []) {
          ]
       );
 
-      $list_windows = ['Windows 3.1', 'Windows 95', 'Windows 98', 'Windows 2000 Pro',
```

Comment:
rather than to have a fixed list of Windows OS, it is necessary to load the list of the windows OS available in GLPI to make sure you don't forget